tyr/autoscaler/autoscaler.py

```diff
@@ -0,0 +1,115 @@
+from boto.ec2.autoscale import AutoScaleConnection
+from boto.ec2.autoscale import LaunchConfiguration
+from boto.ec2.autoscale import AutoScalingGroup
+import boto.ec2
+import logging
+
+
+class AutoScaler(object):
+    '''
+    Autoscaler setup class
+    '''
+
+    log = logging.getLogger('Clusters.AutoScaler')
+    log.setLevel(logging.DEBUG)
+    ch = logging.StreamHandler()
+    ch.setLevel(logging.DEBUG)
+    formatter = logging.Formatter(
+        '%(asctime)s [%(name)s] %(levelname)s: %(message)s',
+        datefmt='%H:%M:%S')
+    ch.setFormatter(formatter)
+    log.addHandler(ch)
```

Comment:
You're going to run into issues with multiple log handlers on the same logger. I'd recommend checking for existing handlers before adding a new one or, better yet, make use of the `establish_logger` method in `tyr.servers.server`.